osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderTail.cs

```diff
@@ -40,7 +46,34 @@ public DrawableSliderTail(Slider slider, SliderTailCircle hitCircle)
             positionBindable.BindValueChanged(_ => updatePosition());
             pathVersion.BindValueChanged(_ => updatePosition(), true);
 
-            // TODO: This has no drawable content. Support for skins should be added.
+            InternalChildren = new Drawable[]
+            {
+                scaleContainer = new Container
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Origin = Anchor.Centre,
+                    Anchor = Anchor.Centre,
+                    Children = new Drawable[]
+                    {
+                        CirclePiece = new SkinnableDrawable(new OsuSkinComponent(OsuSkinComponents.SliderTail), _ => null),
+                    }
+                },
+            };
+        }
+
+        [osu.Framework.Allocation.BackgroundDependencyLoader]
```

Comment:
What's with the fully qualified name? Should just be able to import the namespace.